test/test_view_isodoses_tab.py

```diff
@@ -0,0 +1,109 @@
+import os
+import pytest
+from pathlib import Path
+
+from src.Controller.GUIController import MainWindow
+from src.Model.PatientDictContainer import PatientDictContainer
+from src.View.mainpage.IsodoseTab import isodose_percentages
+from src.Model import ImageLoading
+
+from pydicom import dcmread
+from pydicom.errors import InvalidDicomError
+
+
+def find_DICOM_files(file_path):
+    """
+    Function to find DICOM files in a given folder.
+    :param file_path: File path of folder to search.
+    :return: List of file paths of DICOM files in given folder.
+    """
+
+    dicom_files = []
+
+    # Walk through directory
+    for root, dirs, files in os.walk(file_path, topdown=True):
+        for name in files:
+            # Attempt to open file as a DICOM file
+            try:
+                dcmread(os.path.join(root, name))
+            except (InvalidDicomError, FileNotFoundError):
+                pass
+            else:
+                dicom_files.append(os.path.join(root, name))
+    return dicom_files
+
+
+class TestIsodosesTab:
+    """
+    Class to set up the OnkoDICOM main window for testing the
+    structures tab.
+    """
+    __test__ = False
+
+    def __init__(self):
+        # Load test DICOM files
+        desired_path = str(Path.cwd().joinpath('test', 'testdata', \
+                                               'DICOM-RT-TEST'))
```

Comment:
```suggestion
        desired_path = Path.cwd().joinpath('test', 'testdata', \
                                               'DICOM-RT-TEST')
```
str is not necessary for path